glide.lock

```diff
@@ -147,8 +151,11 @@ imports:
 - name: gopkg.in/yaml.v2
   version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
 testImports:
-- name: github.com/stretchr/testify
-  version: ""
-  repo: https://github.com/stretchr/testify.git
+- name: github.com/davecgh/go-spew
+  version: 6d212800a42e8ab5c146b8ace3490ee17e5225f9
   subpackages:
-  - assert
+  - spew
+- name: github.com/pmezard/go-difflib
```

Comment:
do you know why it's adding this (and go-spew) to the vendor deps but isn't adding it to the vendor directory?